app/internal/on_this_day_events.py

```diff
@@ -0,0 +1,43 @@
+import json
+from datetime import date, datetime
+from typing import Dict
+
+import requests
+from loguru import logger
+from sqlalchemy import func
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import Session
+from sqlalchemy.orm.exc import NoResultFound
+
+from app.database.models import WikipediaEvents
+
+
+def insert_on_this_day_data(session: Session) -> Dict:
```

Comment:
Use `Depends(get_db)` instead. Also, in the type annotations, add types for the `Dict`'s key and value.